recipes/vigra/build.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+EXTRA_CMAKE_ARGS=""
+if [[ `uname` == 'Darwin' ]];
+then
+    export DYLIB_EXT=dylib
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+    export MACOSX_VERSION_MIN="10.7"
+    export CMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export LINKFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11 -L${LIBRARY_PATH}"
+    EXTRA_CMAKE_ARGS="-DCMAKE_OSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}"
```

Comment:
Ensure we are using `clang` on Mac and that we have C++11 support. Also, ensure that Mac OS 10.7 will be supporte at bare minimum.